.github/workflows/charm-pull-request.yaml

```diff
@@ -48,15 +58,36 @@ jobs:
     needs:
       - linting
 
+  node-size-validation:
+    runs-on: ubuntu-latest
+    name: Input Node Size validation
+    steps:
+      - name: Validate node size variable
+        if: ${{ !contains(fromJSON('["medium", "large", "xlarge"]'), inputs.node-size) }}
+        run: exit 1
+    needs:
+      - linting
+      - unit-test
+
   integration-test:
     name: Integration Tests
-    uses: ./.github/workflows/_charm-tests-integration.yaml
```

Comment:
thought: should the `.github/workflows/_charm-tests-integration.yaml` file be deleted or is it still used somewhere?

Reply:
As explained in the description, I didn't remove this workflow in case we want to switch back to use our setup (e.g. call it directly from charm's repository, if for some reason the `operator-workflows` is broken).